sympy/core/relational.py

```diff
@@ -381,12 +381,15 @@ class Equality(Relational):
 
     is_Equality = True
 
-    def __new__(cls, lhs, rhs=0, **options):
+    def __new__(cls, lhs, rhs=None, **options):
         from sympy.core.add import Add
         from sympy.core.logic import fuzzy_bool
         from sympy.core.expr import _n2
         from sympy.simplify.simplify import clear_coefficients
 
+        if rhs == None:
```

Comment:
You can use `is None` rather than `== None` which is more efficient/reliable.

Reply:
> I've just marked two of my comments resolved. There's one left but it is not a big deal so I'd be happy to merge this as is.

I have replaced `==` with None.